src/spyglass/common/common_behav.py

```diff
@@ -296,3 +296,165 @@ def get_abs_path(cls, key: Dict):
                 f"video file with filename: {video_filename} "
                 f"does not exist in {video_dir}/"
             )
+
+
+@schema
+class PositionIntervalMap(dj.Computed):
+    definition = """
+    -> IntervalList
```

Comment:
Is it worth having a dependence on TaskEpoch to have `epoch` as a primary key as well?

Reply:
Not sure if it makes any use cases trickier.  It would require you to know the epoch at population, which may not be as obvious for derived interval_list_names (?).  Epoch could also have a one:many relationship with interval_list_name, so you'd still need a way to select which to use when searching by epoch